### ISSUE TITLE
Wrong link adress (mangled) when quotes are appended

### ISSUE THREAD

**Issue by amabok** [cezarsa/silver_bird#256]
Everytime a link in a tweet that contains a quote - " - rigth after the link adress, it is missinterpreted and read as the quote is part of the link address.

Example tweet:

“@homensdaluta: ÚLTIMA HORA!!!
Homens da Luta fazem especial "Mudam-se os tempos, mudam-se os Tavares!"
Em exclusivo!! http://t.co/fyrHz3f”

Expected link to be opened:
http://t.co/fyrHz3f

Actual link opened:
http://t.co/fyrHz3f”

**Comment by nloadholtes** [comment]
This bugs me too. I've submitted a pull request that has a fix for this.

**Comment by DrWig** [comment]
Yep, I get this issue allot....it's the only major issue I have with silver bird:
I go past on my way into London! “@BillBailey: Touring views no. 7.. A4 Westbound.. time to unpack them for Christmas http://t.co/k0N5Yiu0”

opens http://t.co/k0N5Yiu0”

instead of 

http://t.co/k0N5Yiu0

**Comment by nloadholtes** [comment]
Yes the original pull request I issued didn't fully address this problem. I submitted a new pull request (Pull Request #326) that has code that addresses this issue.
